recipe/meta.yaml

```diff
@@ -29,13 +29,14 @@ requirements:
     - numpy
     - python >=3.8
     - qtpy
+    - superqt
```

Comment:
Hi @rhoitink, it seems the package is directly importing superqt, so it should be added to dependencies upstream.

No need to release a new package, but would be nice to add it :) so that it is there for the next release